src/param.js

```diff
@@ -336,3 +336,23 @@ class Json extends Param {
   }
 }
 module.exports.Json = (...args) => new Json(...args);
+
+class NumberParam extends Json {
+  constructor(name, { positive = false } = {}, ...args) {
```

Comment:
```suggestion
  constructor(name, { enforcePositive = false } = {}, ...args) {
```
?

Reply:
Things like `min, max` etc can come later

Reply:
`nonNegative: false`

Reply:
Let's just replace this with min max